draft-ietf-quic-multipath.md

```diff
@@ -381,6 +381,12 @@ that address. After receiving packets from the
 client on a new path, if the server decides to use the new path,
 the server MUST perform path validation ({{Section 8.2 of QUIC-TRANSPORT}})
 unless it has previously validated that address.
+A server can also decide to not perform path validation if it wants to deny the new path,
+or not perform it immediately if, e.g., no connection ID is yet available.
```

Comment:
Should this also call out the impact of delaying path validation? For example, are anti-amplification limits in place until the path is validated?

Reply:
This likely needs some editorial work anyway based on the discussion in issue #390. I proposed to merge this PR first as it is and then work on another separate PR for issue #390.

However, @MikeBishop what do you mean by "impact on delaying path validation"? Yes, you can't send before the path is validated, or at least not more the amplification limit. But I guess that's the intention. I guess the only thing we can mention is that if you delay longer than the time-out, the other end will assume validation failed and close the path. But that should be fine as well because it will send an explicit abandon.